Extensions/shuffle+.js

```diff
@@ -338,7 +338,7 @@
 		if (errors) throw errors[0].message;
 		if (data.albumUnion.playability.playable === false) throw "Album is not playable";
 
-		return data.albumUnion.tracks.items.filter(({ track }) => track.playability.playable).map(({ track }) => (includeMetadata ? track : track.uri));
+		return data.albumUnion.tracksV2.items.filter(({ track }) => track.playability.playable).map(({ track }) => (includeMetadata ? track : track.uri));
```

Comment:
is not available in all supported spotify versions.
```suggestion
		return (data.albumUnion?.tracksV2 ?? data.albumUnion?.tracks).items.filter(({ track }) => track.playability.playable).map(({ track }) => (includeMetadata ? track : track.uri));
```